src/styles/blocks/card.scss

```diff
@@ -0,0 +1,125 @@
+@import "../partials/variables.scss";
+
+.card {
+  max-width: 198px;
+  min-height: 406px;
+  transition: transform 300ms;
+  &:hover {
+    transform: scale(1.2);
+    transition: transform 300ms ease;
+  }
+  &__content {
+    width: $card__content-width;
+    border: 1px solid $card-border-color;
+    border-radius: 5px;
+    box-sizing: border-box;
+    padding-block: auto;
```

Comment:
There is no such thing as `padding: auto` padding does not have an `auto` value, you can check that in the dev tools.